### ISSUE TITLE
Icon request: Fence

### ISSUE THREAD

**Issue by mittalyashu** [feathericons/feather#697]
## Icon Request

* Icon name: Fence

| Suggestion |
| --- |
| ![image](https://user-images.githubusercontent.com/29014463/61707766-d8fa5800-ad68-11e9-9b28-0875c65213a5.png) |

**Comment by johnletey** [comment]
@mittalyashu I like Suggestion 1 ... in Suggestion 2, the fence posts are two tall!

**Comment by mittalyashu** [comment]
Yeah, I created this icon with the intention to create a fence, but does it really look like one?

**Comment by johnletey** [comment]
@mittalyashu I think so ... remove the dots though

What do you think @locness3 @ahtohbi4?

**Comment by mittalyashu** [comment]
If you remove the dots, it will look like a tower with a badge.

**Comment by johnletey** [comment]
I see ... then keep the dots

**Comment by locness3** [comment]
I prefer the first one
